test/unit-complete/main.tf

```diff
@@ -66,99 +81,126 @@ module "test" {
       revision_name   = "first"
       percent         = 69
       latest_revision = false
+    },
+    {
+      percent         = 31
+      latest_revision = true
+    }
+  ]
 
-      metadata = {
-        namespace = "unit-complete-ns"
-        labels = {
-          foo = "bar"
-        }
-        generation       = 0
-        resource_version = "0.0.0"
-        self_link        = "path/to/self"
-        uid              = "fd32528f-8020-4f04-9e5e-c89ea4f904dd"
-        annotations = {
-          foo = "bar"
-        }
+  metadata = {
+    namespace = "unit-complete-ns"
+    labels = {
+      foo = "bar"
+    }
+    annotations = {
+      foo = "bar"
+    }
+  }
+
+  domain_mapping = {
+    spec = {
+      force_override   = true
+      certificate_mode = "NONE"
+    }
+    metadata = {
+      namespace = "unit-complete-ns"
+      labels = {
+        foo = "bar"
       }
-      domain_mapping = {
-        spec = {
-          force_override   = true
-          certificate_mode = "NONE"
-        }
-        metadata = {
-          namespace = "unit-complete-ns"
-          labels = {
-            foo = "bar"
-          }
-          generation       = 0
-          resource_version = "0.0.0"
-          self_link        = "path/to/self"
-          uid              = "fd32528f-8020-4f04-9e5e-c89ea4f904dd"
-          annotations = {
-            foo = "bar"
-          }
-        }
+      annotations = {
+        foo = "bar"
       }
     }
-  ]
+  }
+
+  computed_members_map = {
+    myserviceaccount = "serviceAccount:${module.test-sa.service_account.email}"
+  }
 
   # add all optional arguments that create additional/extended resources
 
   iam = [
     {
-      role    = "roles/run.invoker"
-      members = ["domain:example-domain"]
+      role = "roles/run.invoker"
+      members = [
+        "allUsers",
+        "allAuthenticatedUsers",
+        "user:user@${local.org_domain}",
+        "serviceAccount:service-account-name@${local.project_id}.iam.gserviceaccount.com",
+        "group:group@${local.org_domain}",
+        "domain:${local.org_domain}",
+        "projectOwner:${local.project_id}",
+        "projectEditor:${local.project_id}",
+        "projectViewer:${local.project_id}",
+        "computed:service-account-name@${local.project_id}.iam.gserviceaccount.com"
```

Comment:
that's not how computed works... you need to use the key of the desired entry in the map you provide.

Reply:
hahaha, I had a momentary lapse of reasoning when did the computed members. Thanks for spotting 🙏🏻